compiler/ilgen/OMRIlBuilder.hpp

```diff
@@ -241,10 +243,16 @@ class IlBuilder : public TR::IlInjector
    TR::IlValue *ConstAddress(const void * const value);
    TR::IlValue *ConstString(const char * const value);
 
-   TR::IlValue *Const(int8_t value)             { return ConstInt8(value); }
-   TR::IlValue *Const(int16_t value)            { return ConstInt16(value); }
-   TR::IlValue *Const(int32_t value)            { return ConstInt32(value); }
-   TR::IlValue *Const(int64_t value)            { return ConstInt64(value); }
+
+   template <typename T>
```

Comment:
Would you consider moving this change into a separate commit at this point, since it doesn't contribute to the main change you're making at this point?

Reply:
A separate commit in this PR or a new PR?

Reply:
sorry, missed the question....either would have been fine, but since it's not related to the PR, feels like a separate PR is most appropriate.